examples/demos/client_merge_demo/main.go

```diff
@@ -47,7 +50,7 @@ func showClientMerge() error {
 	getResp, err := c.SyntheticsAdmin.ListAgents(context.Background(), &syntheticspb.ListAgentsRequest{})
 
 	fmt.Println("Received result:")
-	demos.PrettyPrint(getResp)
+	demos.PrettyPrint(getResp.GetAgents())
```

Comment:
It would be good to show difference (or absence of difference) between results received via v5 and v6 API.

Reply:
`result` is a slice of Users and `getResp.GetAgents()` is a slice of Agents. Those are entirely different objects.

Reply:
Well, I probably then do not understand what this example demonstrates ... but that does not need to hold up the merge ;).